torchinductor/codecache.py

```diff
@@ -138,6 +138,7 @@ def load(cls, source_code):
                 try:
                     subprocess.check_output(cmd, stderr=subprocess.STDOUT)
                 except subprocess.CalledProcessError as e:
+                    print(f"SUBPROCESS ERROR: {' '.join(cmd)} ")
```

Comment:
Makes it a tad easier to just copy the command out of here.

Reply:
`exc.CppCompileError` will print out the command in its message